content/platform/environment.md

```diff
@@ -0,0 +1,17 @@
++++
+title = "Platform Environments"
+weight = 2
+chapter = false
+pre = ""
++++
+
+The Environment in core platform represents the deployment and testing context for a tenant. Each tenant must be associated with an environment where applications can be tested and deployed, allowing teams to manage and isolate different environments efficiently.
```

Comment:
Let's always use the full Platform Environment, may seem overkill but we need to really distinguish between a platform environment e.g. gcp-dev and application environment (functional, nft, integrated).

I think this first line should be "A Platform environment is the underlying infrastructure for Application environments". It scales up and down automatically based on the applications deployed to it. Dev and prod are isolated to reduce the blast radius of platform deployments.